test/karma.conf.js

```diff
@@ -1,10 +1,12 @@
 // Karma configuration
 
-module.exports = function(config, options) {
+module.exports = function(config, overrideOptions) {
     'use strict';
 
+    var options;
+
     // If not being overridden, then specify the default headless configuration
-    if (!options) {
+    if (!overrideOptions) {
```

Comment:
Nit: I completely realize you're just trying to get the styles to pass, so please don't feel obligated to implement this suggestion.  I'd consider changing this `if` statement to the positive: 
```
if (overrideOptions) {
    options = overrideOptions;
} else {
...
}
```

Also, couldn't we do something like [defaults](http://underscorejs.org/#defaults)?